api/v2/models/comment.js

```diff
@@ -0,0 +1,18 @@
+import users from './users';
+
+const author = users.find((user) => user.id);
+
+const comments = [
+  {
+    comment_id: 1,
+    authorId: author.id,
+    comment: 'good article',
+  },
+  {
+    comment_id: 2,
+    authorId: author.id + 1,
+    comment: 'awesome article',
+  },
+];
+
+export default comments;
```

Comment:
Is this file still needed? If yes, for what?

Reply:
thanks for the feedback i am working on it